src/modules/cloudinary/services/cloudinary.service.ts

```diff
@@ -0,0 +1,42 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { UploadApiErrorResponse, UploadApiResponse, v2 } from 'cloudinary';
+import toStream = require('buffer-to-stream');
+
+@Injectable()
+export class CloudinaryService {
+    async uploadImage(
```

Comment:
Since we do not expect to use this method externally I think it makes sense to mark it as private. As well as "removeImages". And please move them below public methods